### PR TITLE
Added customer session to the shipping hook due to issues related to …

### PR DESCRIPTION
**Pull request by johnmccombs1**
…reward points needing the customer's session

### CODE REVIEW

File: app/code/community/Bolt/Boltpay/Helper/Data.php [hunk 1]
```diff
@@ -336,4 +336,27 @@ public function getItemImageUrl($item)
...
+     *
+     * @param $customerId
+     */
+    public function setCustomerSessionById($customerId)
+    {
+        if ($customerId) {
+            Mage::getSingleton('customer/session')->loginById($customerId);
+        }
+    }
 }
```
Comment (YevhenShBolt) [inline app/code/community/Bolt/Boltpay/Helper/Data.php]: Missed new line after }

Comment (johnmccombs1) [reply app/code/community/Bolt/Boltpay/Helper/Data.php]: @YevhenShBolt can you explain this a little more? I don't typically add a new line at the end of my classes. Am I missing something?

Comment (YevhenShBolt) [reply app/code/community/Bolt/Boltpay/Helper/Data.php]: I just noticed when i've reviewed PR that GitHub add a warning here https://github.com/BoltApp/bolt-magento1/pull/206/files. The GitHub said that there is no line at the end of a file.  

We can easily fix it:
- https://help.github.com/articles/dealing-with-line-endings/
- https://stackoverflow.com/a/46325278

**Comment by YevhenShBolt** [comment]
Also, for Magento 2 it will be useful to configure the Sniffers in PHPStorm: https://devdocs.magento.com/guides/v2.2/coding-standards/code-standard-sniffers.html

**Pull request merged**